pydy/viz/scene.py

```diff
@@ -24,24 +25,20 @@
 
 try:
     import IPython
-    from IPython.html import widgets
-    from IPython.display import display, Javascript
-
-    ipy_ver = parse_version(IPython.__version__)
-    if ipy_ver < parse_version('2.0'):
-        raise ImportWarning('PyDy supports IPython >= 2.0.0, while you have ' +
-                            'IPython ' + IPython.__version__ + ' installed. ' +
-                            'IPython related functionalities will not be ' +
-                            'available')
-        ipython_less_than_3 = None
-    elif ipy_ver >= parse_version('2.0') and ipy_ver < parse_version('3.0'):
+    if parse_version(IPython.__version__) < parse_version('3.0'):
+        msg = ('PyDy only supports IPython >= 3.0.0. You have '
+               'IPython {} installed. IPython related functionalities will '
+               'not be available')
+        with warnings.catch_warnings():
```

Comment:
By default Python does not show warnings and I only want these warnings to show once per python session. So if you happen to import this multiple times in single session it will only issue the import warning once.

`catch_warnings` is a nice way to set the filters on a per warning level. I'd also be fine with setting the warning filter to `once` globaly for PyDy. If you have a suggestion of a better pattern for per warning filter control, I can change it.